### DISCUSSION TITLE
How can we configure markdownlint to ignore a file (CHANGELOG.md) from linting ?

### DISCUSSION THREAD

**Discussion by aloknj** [super-linter/super-linter#2359] [Ideas]
In our scenario, we want to skip or ignore a .md file from markdownlint. 

Looking at:
.github/linters/.markdown-lint.yml

I was wondering whether this is possible or not. If possible how to add that in this file.
For example, in our case we want **to disable `MD012` rule for CHANGELOG.md file** only and this rule shall be enabled for other .md files. 

Thanks for looking into it

**Comment by nvuillam** [comment]
You can disable a rule directly in the markdown file :)

https://github.com/DavidAnson/markdownlint#configuration
